source/module_esolver/esolver_ks_pw.cpp

```diff
@@ -586,7 +586,17 @@ void ESolver_KS_PW<T, Device>::hamilt2density(const int istep,
         this->phsol->solve(this->p_hamilt,      // hamilt::Hamilt<T, Device>* pHamilt,
             this->kspw_psi[0],   // psi::Psi<T, Device>& psi,
             this->pelec,         // elecstate::ElecState<T, Device>* pelec,
-            GlobalV::KS_SOLVER); // const std::string method_in,
+            GlobalV::KS_SOLVER, // const std::string method_in,
```

Comment:
Since qianrui has designed new input module, could you change GlobalV::xxx to PARAM.inp.xxx?